Detectors/ITSMFT/common/reconstruction/src/AlpideCoder.cxx

```diff
@@ -13,9 +13,29 @@
 
 #include "ITSMFTReconstruction/AlpideCoder.h"
 #include <TClass.h>
+#include <TFile.h>
 
 using namespace o2::itsmft;
 
+NoiseMap* AlpideCoder::mNoisyPixels = nullptr;
+int AlpideCoder::mNoiseThreshold = 3;
+
+//______________________________________________________________
+void AlpideCoder::loadNoisyPixels(const std::string& noise)
+{
+  TFile f(noise.data(), "old");
+  if (!f.IsOpen()) {
+    LOG(ERROR) << "Cannot load noisy pixels from " << noise << '\n';
+    return;
+  }
+
+  LOG(INFO) << "Loading noisy pixels from " << noise << '\n';
+
+  auto pnoise = (std::vector<std::map<int, int>>*)f.Get("Noise");
+  mNoisyPixels = new NoiseMap(*pnoise);
```

Comment:
the map can be quite large, why do you need to copy it, better simply assign the pointer of the object from the file.

Reply:
@shahor02 Actually, the map here is not copied. It is swapped.  If it is OK, of course.

Reply:
swapping is better, but still, I would prefer to not modify ccdb objects, as this invalidates the cache.

Reply:
@shahor02  OK. I will see how to keep that pointer directly of the root file.